Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-49704_and_CVE-2025-49706_and_CVE-2025-53770_and_CVE-2025-53771_-_Microsoft_SharePoint_ToolShell_vulnerability_chain.yml

```diff
@@ -0,0 +1,3317 @@
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "38"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3583.75,
+          "y": 3974
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "92":
+    id: "92"
+    taskid: 0464b840-88f6-479c-94ca-86bdbe0081d5
+    type: regular
+    task:
+      id: 0464b840-88f6-479c-94ca-86bdbe0081d5
+      version: -1
+      name: Set agent IDs to alert context
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setAlert
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      agentsid:
+        simple: ${PossibleSPServers.results.agent_id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2950.25,
+          "y": 823
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "93":
+    id: "93"
+    taskid: 4574151e-8f13-4668-8a01-6f18fe967836
+    type: condition
+    task:
+      id: 4574151e-8f13-4668-8a01-6f18fe967836
+      version: -1
+      name: Are There Any Suspicious Findings
+      description: Are there any potential WebShells found?
```

Comment:
What about the related alerts?

Reply:
Shouldn't we show the actual finding?

Reply:
Great catch - added this to the condition 

As for the second question, I've small objects from the findings in the following manual action ( and pointed to the massive results, if there will be, in the relevant tasks.